ios/Classes/SwiftFlutterCrispChatPlugin.swift

```diff
@@ -26,7 +26,7 @@ public class SwiftFlutterCrispChatPlugin: NSObject, FlutterPlugin, UIApplication
             if(crispConfig.tokenId != nil){
                 CrispSDK.setTokenID(tokenID: crispConfig.tokenId!)
             }if(crispConfig.sessionSegment != nil){
-                CrispSDK.setTokenID(tokenID: crispConfig.sessionSegment!)
+                CrispSDK.setSessionSegment(tokenID: crispConfig.sessionSegment!)
```

Comment:
As per crisp iOS documentation this need to be

`CrispSDK.session.segment = crispConfig.sessionSegment!` — Sets session segment